src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -52,7 +52,7 @@
   private static final String[] STUN_SERVERS = {
       "numb.viagenie.ca", "stun.ekiga.net", "stun.gmx.net", "stun.1und1.de"};
   private static final int STUN_SERVER_PORT = 3478;
-  private static final int LOCAL_FORWARDING_PORT = 2601;
+  private static final int PROXY_PORT = 2601;
```

Comment:
Run into a problem:
class ReliableSocketProfile.java has a constant "MAX_SEGMENT_SIZE" which access a method called "Rscc.getPackageSize()" <-- since this method is static, I can't remove it.
Does ReliableSocketProfile.java needs to have an up-to-date value, if it changes in the GUI? Then we need to refactor a bit more